### ISSUE TITLE
Stats: Subscribers - add chart tooltip

### ISSUE THREAD

**Issue by grzegorz-cp** [Automattic/wp-calypso#75936]
Until the ticket can be scoped, we need more guidance if the tooltip replaces the chart's dynamic legend or if they both could show the same information.

**Comment by jsnmoon** [comment]
Let's go ahead and shelve this task for now since the dynamic legend does everything we would need the tooltip to do. In fact, the dynamic legend might do an even better job handling multi-series charts than a typical tooltip. What do you think?

**Comment by grzegorz-cp** [comment]
Yes, the existing legend has more potential to display different data sets and include links to redirect people to some details pages.
I'll move the task to our backlog.
